sdk/identity/azure-identity/tests/test_environment_credential.py

```diff
@@ -21,12 +21,13 @@
 
 
 def test_error_message():
-    """get_token should raise CredentialUnavailableError for incomplete configuration, listing any set variables."""
+    """get_token should raise CredentialUnavailableError for incomplete configuration."""
 
     with mock.patch.dict(os.environ, {}, clear=True):
         with pytest.raises(CredentialUnavailableError) as ex:
             EnvironmentCredential().get_token("scope")
-    assert not any(var in ex.value.message for var in ALL_VARIABLES)
+    assert "EnvironmentCredential authentication unavailable. "
```

Comment:
I suggest removing these asserts because the string isn't interesting to test now that it's static and copying it here creates a low value maintenance burden.